client/pause.go

```diff
@@ -0,0 +1,44 @@
+package client
+
+import (
+	"fmt"
+	"github.com/hyperhq/hyper/engine"
+	"net/url"
+	"strings"
+
+	gflag "github.com/jessevdk/go-flags"
+)
+
+func (cli *HyperClient) HyperCmdPause(args ...string) error {
+	var parser = gflag.NewParser(nil, gflag.Default|gflag.IgnoreUnknown)
+	parser.Usage = "pause CONTAINER\n\nPause the container"
+	args, err := parser.Parse()
+	if err != nil {
+		if !strings.Contains(err.Error(), "Usage") {
+			return err
+		} else {
+			return nil
+		}
+	}
+	if len(args) == 1 {
+		return fmt.Errorf("Can not accept the 'pause' command without Container ID!")
+	}
+
+	v := url.Values{}
+	v.Set("container", args[1])
```

Comment:
Already make sure args array at least has one arg. Doesn't line 23 did this job? or we need another check here?